iroh-relay/src/client.rs

```diff
@@ -37,6 +37,48 @@ pub(crate) mod streams;
 #[cfg(not(wasm_browser))]
 mod util;
 
+/// Client related errors
+#[allow(missing_docs)]
```

Comment:
I don't think there is much value in documenting these variants currently, which is why I am skipping them